pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -141,7 +141,7 @@ in rec {
 
         ${workspaceDependencyLinks}
 
-        yarn install ${lib.escapeShellArgs (defaultYarnFlags ++ yarnFlags)}
+        yarn install ${lib.escapeShellArgs (defaultYarnFlags ++ lib.optional "--ignore-scripts" ++ yarnFlags)}
```

Comment:
This looks wrong, should it be `defaultYarnFlags ++ (lib.optionals ignoreScripts ["--ignore-scripts"]) ++ yarnFlags`?

Reply:
``lib.optionals ignoreScripts ["ignore-scripts"]`` and lib.optional ignoreScripts "--ignore-scripts"`` is the same thing and you can chain lib.optional without ``()``.

Reply:
Oh, you're right. Ommiting () only works when assigning to a var, not as function argument.

Reply:
I forgot the first argument 🤦🏼